packer/progressbar.go

```diff
@@ -15,26 +15,28 @@ func ProgressBarConfig(bar *pb.ProgressBar, prefix string) {
 	bar.Prefix(prefix)
 }
 
-var defaultUiProgressBar = &uiProgressBar{}
+var defaultUiProgressBar = &UiProgressBar{}
```

Comment:
If we stop using the `defaultUiProgressBar` then we could get rid of this global variable; it was meant to be a singleton so that multi downloads wouldn't fight over the ui.
```suggestion
```